src/components/widgets-common/top-buttons/top-buttons.directive.coffee

```diff
@@ -16,6 +16,20 @@ module.controller('CommonTopButtonsCtrl', ($scope, $rootScope, $log, ImpacWidget
 
   $scope.hasInfo = ->
     w && w.content? && w.content.info? && w.content.info.length > 0
+
+  $scope.isCsvExportable = ->
+    templates = _.filter(ImpacDashboardsSvc.getWidgetsTemplates(), {name: w.name})
+    templates.length > 0 && templates[0].isBolt
```

Comment:
Rather than adding the `isBolt` property to the template objects. I think you could just go:
```
templates.length > 0 && _.get(template, 'metadata.bolt_path')
```